getting-started/v2-v4-upgrade.md

```diff
@@ -43,11 +43,11 @@ Here is an example of a `package.json` with all the changes:
     "@shipengine/connect-runtime": "^4.0.0"
   },
   "devDependencies": {
-    "@shipengine/connect-local": "^4.0.0"
+    "@shipengine/connect": "^4.3.0"
```

Comment:
In PR https://github.com/shipstation/ipaas-connect/pull/892/files there's 4.2.0 version, if that matters.